MERGE_CONFLICTS.md

```diff
@@ -45,11 +51,33 @@ but these tips can help reduce the likelihood of pain here:
   your changes relate to and will affect changes other people have made,
   Badness is very likely to ensue.
```

Comment:
Some of the advice here is less critical now that we have branch protection, but I think it's all still good advice.